spec/progress_bar_characters_spec.rb

```diff
@@ -0,0 +1,44 @@
+require File.expand_path(File.join(File.dirname(__FILE__), 'spec_helper'))
+
+describe 'ProgressBar bar output with custom characters' do
+  before do
+    Timecop.freeze Time.utc(2010, 3, 10, 0, 0, 0)
+    @progress_bar = ProgressBar.new(100, bar: '█', delimiters: '▕▏')
```

Comment:
Oh, there is kind of a test for multibyte characters I guess. Why are there two separate spec files?

Reply:
I wasnt certain the `arguments_spec` really covered the use case. I was more of an interface positive negative check. I wanted a test that specifically checked the feature and its output